clang/lib/DPCT/APINames.inc

```diff
@@ -361,7 +361,7 @@ ENTRY(atomicExch, atomicExch, true, NO_FLAG, P0, "Successful")
 ENTRY(atomicMin, atomicMin, true, NO_FLAG, P0, "Successful")
 ENTRY(atomicMax, atomicMax, true, NO_FLAG, P0, "Successful")
 ENTRY(atomicInc, atomicInc, true, NO_FLAG, P0, "Successful: helper function")
-ENTRY(atomicDec, atomicDec, false, NO_FLAG, P0, "no direct mapping yet")
+ENTRY(atomicDec, atomicDec, true, NO_FLAG, P0, "Successful: helper function")
```

Comment:
"Successful: helper function", Just a ask, should be filled in "Successful: helper function" in the future when using help function as API mapping.

Reply:
I think we need a consistent format of MigrationDesc

Reply:
Will update them with "Successful".